examples/Notebook/Notebook Basics.ipynb

```diff
@@ -11,57 +11,125 @@
+    "You can start the notebook server from the command line (Terminal on Mac/Linux, CMD prompt on Windows) by running the following command: \n",
     "\n",
     "    ipython notebook\n",
     "\n",
-    "For more details on how to run the notebook server, see [Running the Notebook Server](Running the Notebook Server.ipynb)."
+    "This will print some information about the notebook server in your terminal, including the URL of the web application (by default, `http://127.0.0.1:8888`). It will then open your default web browser to this URL.\n",
+    "\n",
+    "When the notebook opens, you will see the **notebook dashboard**, which will show a list of the notebooks and subdirectories in the directory where the notebook server was started (as seen in the next section, below). Most of the time, you will want to start a notebook server in the highest directory in your filesystem where notebooks can be found. Often this will be your home directory."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## The Notebook dashboard"
+    "### Additional options"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "When you first start the notebook server, your browser will open to the notebook dashboard. The dashboard serves as a home page for the notebook. Its main purpose is to display the notebooks in the current directory. For example, here is a screenshot of the dashboard page for the `examples` directory in the IPython repository:\n",
+    "You can start more than one notebook server at the same time. By default, the first notebook server starts on port 8888 and later notebook servers search for open ports near that one.\n",
     "\n",
-    "<img src=\"images/dashboard_notebooks_tab.png\" />\n",
+    "You can also specify the port manually:\n",
     "\n",
-    "The top of the notebook list displays clickable breadcrumbs of the current directory. By clicking on these breadcrumbs or on sub-directories in the notebook list, you can navigate your file system.\n",
+    "    ipython notebook --port 9999\n",
     "\n",
-    "To create a new notebook, click on the \"New Notebook\" button at the top of the list.\n",
+    "Or start notebook server without opening a web browser.\n",
     "\n",
-    "Notebooks can be uploaded to the current directory by dragging a notebook file onto the notebook list or by the \"click here\" text above the list.\n",
+    "    ipython notebook --no-browser\n",
     "\n",
-    "The notebook list shows a red \"Shutdown\" button for running notebooks and a \"Delete\" button for stopped notebooks. Notebook remain running until you explicitly click the \"Shutdown\" button; closing the notebook's page is not sufficient.\n",
+    "The notebook server has a number of other command line arguments that can be displayed with the `--help` flag: \n",
     "\n",
-    "To see all of your running notebooks along with their directories, click on the \"Running\" tab:\n",
```

Comment:
No the `You can start more than one notebook server at the same time. By default, the first ...` and all this stuff about port. This was mostly, IMHO, when people were using many servers for many dirs and/or languages.